pkg/entities/optimizely_config_test.go

```diff
@@ -0,0 +1,100 @@
+/****************************************************************************
+ * Copyright 2019, Optimizely, Inc. and contributors                        *
+ *                                                                          *
+ * Licensed under the Apache License, Version 2.0 (the "License");          *
+ * you may not use this file except in compliance with the License.         *
+ * You may obtain a copy of the License at                                  *
+ *                                                                          *
+ *    http://www.apache.org/licenses/LICENSE-2.0                            *
+ *                                                                          *
+ * Unless required by applicable law or agreed to in writing, software      *
+ * distributed under the License is distributed on an "AS IS" BASIS,        *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+ * See the License for the specific language governing permissions and      *
+ * limitations under the License.                                           *
+ ***************************************************************************/
+
+package entities
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+)
+
+type OptimizelyConfigTestSuite struct {
+	suite.Suite
+	featureList              []Feature
+	experimentList           []Experiment
+	expectedOptimizelyConfig OptimizelyConfig
+}
+
+var featurelist = `[{"ID":"4482920077","Key":"mutex_group_feature","FeatureExperiments":[{"AudienceIds":[],"ID":"12198292373","LayerID":"12235440722","Key":"experiment_4000",
```

Comment:
Why is this necessary? Is this the same contents from the json test files? Can't we just load those and reuse as json objects?

Reply:
this is not exactly the same contents. These are just feature list and experiment list. This is used just to perform unit tests here (to keep the coverage). I am testing the whole thing again on project config level.